conversions/binary_to_octal.dart

```diff
@@ -0,0 +1,42 @@
+//Author:VishnuPothan
```

Comment:
You could add some tests
```suggestion
import 'package:test/test.dart';

//Author:VishnuPothan
```